controller/cmd/service-mirror/cluster_watcher.go

```diff
@@ -888,10 +896,24 @@ func (rcsw *RemoteClusterServiceWatcher) handleConsiderGatewayUpdateDispatch(eve
 func (rcsw *RemoteClusterServiceWatcher) createOrUpdateService(service *corev1.Service) error {
 	localName := rcsw.mirroredResourceName(service.Name)
 	localService, err := rcsw.localAPIClient.Svc().Lister().Services(service.Namespace).Get(localName)
-	gtwData := getGatewayMetadata(service.Annotations)
+	var localNotFound = false
```

Comment:
nit: why `var` declaration here vs `localNotFound := false` (as above)?

Reply:
also, tioli, but this would be maybe clearer as `localMissing := false`...

having to reason through double-negation and what is actually meant is a little tricky... I wouldn't say that `localFound := true` -- which is what is implied by `localNotFound := false`...